public/app/features/alerting/unified/api/buildInfo.ts

```diff
@@ -162,6 +185,21 @@ export async function fetchPromBuildInfo(url: string): Promise<PromBuildInfoResp
   return response?.data;
 }
 
+export async function fetchPromBuildInfoFallback(url: string): Promise<PromBuildInfoSeriesResponse | undefined> {
+  const response = await lastValueFrom(
+    getBackendSrv().fetch<PromBuildInfoSeriesResponse>({
+      url: `${url}/api/v1/query?query=prometheus_build_info`,
+      showErrorAlert: false,
+      showSuccessAlert: false,
+    })
+  ).catch((e) => {
+    // We have failed to get the version number
+    return undefined;
```

Comment:
We clearly need better error handling here, but we don't want to show anything in the UI if this fails. Maybe log a warning in the console?